src/auth/auth.controller.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Controller,
+  Request,
+  Post,
+  Get,
+  UseGuards,
+  Body,
+} from '@nestjs/common';
+import { ApiOperation, ApiResponse, ApiBearerAuth } from '@nestjs/swagger';
+import { LocalAuthGuard } from './guards/local-auth.guard';
+import { AuthService } from './auth.service';
+import { CreateUserDto } from '../users/create-user.dto';
+import { SignInDto } from './signin.dto';
+import { JwtAuthGuard } from './guards/jwt-auth.guard';
+
```

Comment:
```suggestion
@UseInterceptors(ClassSerializerInterceptor)
```

We should add this decorator for the @Exclude decorator work in the User Entity
https://docs.nestjs.com/techniques/serialization#exclude-properties